File: Vibe.Domain/Employees/Employee.cs
namespace Vibe.Domain.Employees
{
    public class Employee
    {
        public Guid Id { get; }
        public String Name { get; }
        public String Phone { get; }
        public String Login { get; }
        public String Email { get; }

        public Employee(Guid id, String name, String phone, String login, String email)
        {
            Id = id;
            Name = name;
            Phone = phone;
            Login = login;
            Email = email;
        }
    }
}

File: Vibe.BackOffice/vibe.backoffice.client/src/domain/employees/employee.ts
export class Employee {
    constructor(
        public id: string,
        public name: string, 
        public phone: string,
        public login: string,
        public email: string, 
    ) {}

}

export function mapToEmployee(data: any): Employee {
    return new Employee(data.id, data.name, data.phone, data.login, data.email)
}

export function mapToEmployees(data: any[]): Employee[] {
    return data.map(d => mapToEmployee(d))
}